### DISCUSSION TITLE
Looking for assist for "Slot-locking" ???

### DISCUSSION THREAD

**Discussion by mdedmon** [shedaniel/RoughlyEnoughItems#1076] [Q & A] [answered]
I'm running All of Fabric 5 ver 1.3.1 on a server and I've built a very large ME system with auto-crafting and more. It's all working but for one strange recipe confusion. I'm looking for some guidance.  I'll try to outline what's happening and hope someone has some direction.

1.  Start an auto-crafting recipe for stainless steel (no errors, it begins crafting)
2. After noticing it's stuck on stainless steel dust, I check the machine (Stainless Steel Mixer) and it has partial ingredients stuck in the machine.
![Screen Shot 2022-08-14 at 12 41 51 PM](https://user-images.githubusercontent.com/459278/184547284-034dc33d-0de2-48ef-b8f3-f419de483cfc.png)
3. After some troubleshooting, I discover that the machine is actually crafting the invar recipe instead of the stainless steel recipe. I could replicate this by adding the 4th ingredient (nickel dust) to the machine manually and watch it craft Invar.
![Screen Shot 2022-08-14 at 12 44 09 PM](https://user-images.githubusercontent.com/459278/184547282-afca66cb-5f71-4621-a40e-92261914901e.png)
4. The actual stainless steel recipe is this and shares 2 of the same ingredients as the invar receipe:
![Screen Shot 2022-08-14 at 12 42 25 PM](https://user-images.githubusercontent.com/459278/184547283-7ac4dc26-30d3-4393-abb4-9894658daf0b.png)
5. This had me stumped for a while, but I just happened to press [Shift] on the crafting window (as suggested) and saw this message, "Use Slot-Locking with REI to differentiate its recipe from the invar dust"
![Screen Shot 2022-08-14 at 12 41 20 PM](https://user-images.githubusercontent.com/459278/184547285-279eea40-f285-4605-8f96-e5960c55eb2c.png)

OK - So that last message gave me some hope. Some mod (AE2, Modern Industrialization, REI, etc.) knows there is a conflict and has a solution. But, I have no idea how to do this and can't find any documentation or information about this feature.

Any help or suggestions is GREATLY appreciated.

**Comment by shedaniel** [comment, marked as answer]
Sorry this is a discussion forum for REI, and not other mods. You need to find support from AE2 or MI instead.

**Comment by mdedmon** [comment]
Ugh - I get it. You're not the first to give me this same line.

Every discord / support site I've tried, just says, hey, this isn't our
mod. Go somewhere else.

BUT, it clearly says, "Use Slot-Locking with REI..." in the message. Can
you at least confirm, is this a feature of REI? Do you know what
slot-locking is? I just need some help with this.

**Reply by shedaniel** [reply]
No, this isn't a feature in REI. I don't know what slot-locking means at all.

**Comment by mdedmon** [comment]
Ok - thanks for your time. And, fantastic mod BTW.
